switchboard/sbdut.py

```diff
@@ -614,10 +616,43 @@ def simulate(
                     args=plusargs_to_args(plusargs) + args
                 )
 
+        # Add newly created Popen object to subprocess list
+        self.subprocess_list.append(p)
+
         # return a Popen object that one can wait() on
 
         return p
 
+    def terminate(
```

Comment:
you might want to add this to SbNetwork as well?

Reply:
Done.